safe_transaction_service/analytics/views_v2.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework.generics import ListAPIView
+from rest_framework.renderers import JSONRenderer
+from rest_framework.response import Response
+
+from safe_transaction_service.analytics.services.analytics_service import (
+    get_analytics_service,
+)
+
+
+class AnalyticsMultisigTxsByOriginListView(ListAPIView):
+    pagination_class = None
+    swagger_schema = None
+    renderer_classes = (JSONRenderer,)
+
+    def get(self, request, format=None):
+        analytics_service = get_analytics_service()
+        return Response(analytics_service.get_safe_transactions_per_safe_app())
```

Comment:
Maybe you could use a serializer so we get swagger documentation, but it works for me as it is

Reply:
I will do in a different PR.